### ISSUE TITLE
On device training (up stream training)

### ISSUE THREAD

**Issue by StuartIanNaylor** [Ohlomonchick/bcresnet-tflite#1]
No issue just wondering if you had ever done anthing with the now called https://ai.google.dev/edge/litert/models/ondevice_training
TFlite or LiteRT but for a KWS actually training a small dataset to shift the weights to captured usage data could be very beneficial.

Also in https://github.com/google-research/google-research/tree/master/kws_streaming they subclassed keras to provide streaming models even though now I can not seem to get the framework to work without Keras errors but had no problems 2 years ago...

The bcresnet model seems to give great accuracy for very few params and could be the best opensource KWS and wondered if the above was of any interest and maybe implementation as I dunno what the equivalent would be with Torch and then export?
